src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -940,6 +940,54 @@ TEST_CASE("GradientFlexibleReLULayerTest", "[ANNLayerTest]")
 }
 */
 
+/**
+ * Simple ELU module test.
+ */
+TEST_CASE("SimpleELULayerTest", "[ANNLayerTest]")
+{
+    arma::mat output, input, delta;
+    ELU module(0.5);
+    module.Training() = true;
+    module.InputDimensions() = std::vector<size_t>({ 1 });
+    module.ComputeOutputDimensions();
+
+
+    // Test the Forward function.
+    input = arma::mat("-1 2");
+    module.Forward(input, output);
+    REQUIRE(arma::accu(arma::abs(arma::mat("-0.316060 2") - output)) ==
+        Approx(0.0).margin(1e-4));
+
+    // Test the Backward function.
+    module.Backward(input, output, delta);
+    REQUIRE(arma::accu(arma::abs(arma::mat("-0.058136 2") - delta)) ==
+        Approx(0.0).margin(1e-4));
+}
+
+/**
+ * Simple CELU module test.
+ */
+TEST_CASE("SimpleCELULayerTest", "[ANNLayerTest]")
+{
+    arma::mat output, input, delta;
+    CELU module(0.5);
+    module.Training() = true;
+    module.InputDimensions() = std::vector<size_t>({ 1 });
+    module.ComputeOutputDimensions();
+
+
+    // Test the Forward function.
+    input = arma::mat("-1 2");
+    module.Forward(input, output);
+    REQUIRE(arma::accu(arma::abs(arma::mat("-0.432332 2") - output)) ==
+        Approx(0.0).margin(1e-4));
+
+    // Test the Backward function.
+    module.Backward(input, output, delta);
+    REQUIRE(arma::accu(arma::abs(arma::mat("-0.0585098 2") - delta)) ==
+        Approx(0.0).margin(1e-4));
+}
```

Comment:
Were there no commented-out tests that we could enable too?

Reply:
Nope, there weren't any.

Reply:
It seems like there are some in `activation_functions_test.cpp`.  Do you think you could re-enable them?

Also the merge of #3194 makes this a little tedious---for these layers, you can add these tests to `src/mlpack/tests/ann/activation_functions_test.cpp`, and then move the commented-out ones in `src/mlpack/tests/ann/not_adapted/activation_functions_test.cpp` to `src/mlpack/tests/ann/activation_functions_test.cpp` too. :+1:

Reply:
Sorry! Didn't notice them. I'll re-enable them soon.